catalog/spatial/registry/registry-federation-admin-service-impl/pom.xml

```diff
@@ -93,6 +93,13 @@
             <artifactId>filter-proxy</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- Upgrade mockito to version 2.1.0 for this module to get final mock capability -->
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.1.0</version>
```

Comment:
Why aren't we using `{mockito-core.version}` here?

Reply:
If this project, doesn't need to Mock final methods than it should just use the default one.

Reply:
I believe both options could work. This way just focuses on the need to mock final methods.